cSpell.json

```diff
@@ -147,6 +147,13 @@
                 "Accesskey"
             ]
         },
+        {
+          "filename": "**/specification/billing/**/*.json",
+          "words": [
+              "mosp",
```

Comment:
>mosp [](http://example.com/codeflow?start=15&length=4)

whats a mosp?

Reply:
Microsoft Online Services Program (legacy billing model). I believe it is used in an example name